tsconfig.json

```diff
@@ -1,14 +1,14 @@
 {
   "compilerOptions": {
-    "allowSyntheticDefaultImports": true,
+    "outDir": "dist",
     "declaration": true,
     "esModuleInterop": true,
     "forceConsistentCasingInFileNames": true,
-    "module": "commonjs",
+    "module": "es2022",
     "moduleResolution": "node",
```

Comment:
ah right. `nodenext` is used in sign-addon because otherwise it breaks typescript. Of course, when I use `nodenext` in this project, it breaks but for other reasons :D Let me fix that